# Description: Add an Oxide stub for MediaOzonePlatform. There doesn't seem
#  to be any other way to override this
# Author: Chris Coulson <chris.coulson@canonical.com>

diff --git a/media/ozone/media_ozone_platform.cc b/media/ozone/media_ozone_platform.cc
--- a/media/ozone/media_ozone_platform.cc
+++ b/media/ozone/media_ozone_platform.cc
@@ -43,16 +43,20 @@ MediaOzonePlatform* CreateMediaOzonePlat
 MediaOzonePlatform* CreateMediaOzonePlatformGbm() {
   return new MediaOzonePlatformStub;
 }
 
 MediaOzonePlatform* CreateMediaOzonePlatformTest() {
   return new MediaOzonePlatformStub;
 }
 
+MediaOzonePlatform* CreateMediaOzonePlatformOxide() {
+  return new MediaOzonePlatformStub;
+}
+
 MediaOzonePlatform::MediaOzonePlatform() {
   CHECK(!instance_) << "There should only be a single MediaOzonePlatform.";
   instance_ = this;
 }
 
 MediaOzonePlatform::~MediaOzonePlatform() {
   CHECK_EQ(instance_, this);
   instance_ = NULL;
